internal/aws/containerinsight/utils.go

```diff
@@ -166,8 +167,9 @@ func ConvertToOTLPMetrics(fields map[string]any, tags map[string]string, logger
 		if tagKey == Timestamp {
 			timeNs, _ := strconv.ParseUint(tagValue, 10, 64)
 			timestamp = pcommon.Timestamp(timeNs)
```

Comment:
`timestamp` is added to the data point itself, so it still needs to be calculated

Reply:
I am a little worried that this will break the use cases for AWS Container Insights. Have you tested it with AWS Container Insights?

Reply:
Thanks @pxaws - besides the tests in the repo I deployed it, but, in my case I consume the metrics via the OTLP exporter. Do you have a specific test in mind to validate that?

Reply:
A second thought: we remove it, if it causes problems, I can quickly add an option to make it configurable.

Reply:
OK. In this case, you can go ahead.